etc/config/csharp.defaults.properties

```diff
@@ -0,0 +1,17 @@
+compilers=csharp
```

Comment:
Generally, I want to make it a compiler group so compiler explorer can compile code against a variety of versions of dotnet, and make both `targetOS` and `targetArch` configurable via compiler options, but I'd like to make it in another PR.

Reply:
That was something I wanted to address as well. In that respect the compilerProps() calls probably also needs some tweaking. We can do it after this PR definitely.

Reply:
Just got some free time so done in this PR.